src/Model/IndexInfo.php

```diff
@@ -95,9 +97,12 @@ public function getName()
      * Return the index namespace (e.g. "db.collection").
      *
      * @return string
+     * @deprecated Deprecated in 1.7.0
      */
     public function getNamespace()
     {
+        @trigger_error(sprintf('The "%s" method is deprecated since mongodb/mongodb 1.7 and will be dropped in 2.0.', __METHOD__));
```

Comment:
IIRC, we used this approach in other cases where the server stopped supporting an option (e.g. `mapReduce`). In this case, we know the server is no longer supporting this but there's no danger of using this method since we'll still populate the value.

Would it make sense to defer this until the final 1.x release before 2.0, similar to what is done in Doctrine?

Reply:
I've removed the deprecation for now, we can always deprecate in a later release.

Reply:
SGTM. I also just realized that https://github.com/mongodb/specifications/commit/8ed5d562f073f2e43e24c092492fa5ffe1a6b115 doesn't actually require deprecation.